frontend_tests/node_tests/common.js

```diff
@@ -10,6 +10,11 @@ mock_cjs("jquery", $);
 
 const noop = () => {};
 
+mock_cjs("tippy.js", (selector) => {
+    $(selector)[0]._tippy = noop;
+    $(selector)[0]._tippy.setContent = noop;
+});
```

Comment:
`tippy.js` is actually an ES module (see `"module": "dist/tippy.esm.js"` in `node_modules/tippy.js/package.json`) that we default-import, so it should be mocked with `mock_esm`:

```suggestion
mock_esm("tippy.js", {
    default: (selector) => {
        $(selector)[0]._tippy = noop;
        $(selector)[0]._tippy.setContent = noop;
    },
});
```

Reply:
Ah yes, will update.

Reply:
Updated.